caluma/caluma_form/structure.py

```diff
@@ -49,6 +49,8 @@ def get(self, name, default=None):
             out = out()
         if isinstance(out, Element) or isinstance(out, dict):
             return out
+        if out is None:
+            return None
```

Comment:
@winged I still added coverage for the "disconnected document" case - I had to add this line to make it pass, because otherwise the returned value is the string `"None"` instead of `null`.

Reply:
Ah yes makes perfect sense